src/components/Tips.js

```diff
@@ -0,0 +1,134 @@
+import { h } from 'preact'
+import { useRef, useEffect } from 'preact/hooks'
+import { useNavigation, useI18n, useSoftkey, usePopup } from 'hooks'
+import { ListView } from 'components'
+import { goto } from 'utils'
+
+export const Tips = () => {
+  const containerRef = useRef()
+  const listRef = useRef()
+  const i18n = useI18n()
+  const [, setNavigation, getCurrent] = useNavigation('Tips', containerRef, listRef, 'y')
+
+  const [showReadPopup] = usePopup(ReadPopup, { mode: 'fullscreen' })
+  const [showSearchPopup] = usePopup(SearchPopup, { mode: 'fullscreen' })
+  const [showSwitchPopup] = usePopup(SwitchPopup, { mode: 'fullscreen' })
+  const [showAboutWikipediaPopup] = usePopup(AboutPopup, { mode: 'fullscreen' })
+
+  const onKeyCenter = () => {
+    const { index } = getCurrent()
+    const item = items[index]
+    item.action()
+  }
+
+  const onReadPopupSelected = () => {
+    showReadPopup({ onSearchPopupSelected })
+  }
+
+  const onSearchPopupSelected = () => {
+    showSearchPopup({ onSwitchPopupSelected })
+  }
+
+  const onSwitchPopupSelected = () => {
+    showSwitchPopup({ showAboutWikipediaPopup })
+  }
+
+  useSoftkey('Tips', {
+    left: i18n('softkey-back'),
+    onKeyLeft: () => history.back(),
+    center: i18n('centerkey-select'),
+    onKeyCenter,
+    onKeyBackspace: () => history.back()
+  })
+
+  useEffect(() => {
+    setNavigation(0)
+  }, [])
+
+  const items = [
+    { title: i18n('tips-read'), action: onReadPopupSelected },
+    { title: i18n('tips-search'), action: onSearchPopupSelected },
+    { title: i18n('tips-switch'), action: onSwitchPopupSelected },
+    { title: i18n('tips-about'), action: showAboutWikipediaPopup }
+  ]
+
+  return (
+    <div class='tips' ref={containerRef}>
+      <ListView
+        header={i18n('tips-header')}
+        items={items}
+        containerRef={listRef}
+      />
+    </div>
+  )
+}
+
+const tip = (origin, content, close, onNext, onTry) => {
+  const i18n = useI18n()
+  const aboutTip = origin === 'AboutPopup'
+
+  const onTryHandler = () => {
+    if (origin === 'SearchPopup') {
+      close()
+      onTry()
+    } else {
+      onTry(close)
+    }
+  }
+
+  useSoftkey(origin, {
+    left: i18n('softkey-back'),
+    onKeyLeft: close,
+    center: !aboutTip ? i18n('softkey-try') : '',
+    onKeyCenter: !aboutTip ? onTryHandler : null,
+    right: !aboutTip ? i18n('softkey-next') : '',
+    onKeyRight: !aboutTip ? () => { close(); onNext() } : null,
+    onKeyBackspace: close
+  }, [])
```

Comment:
I prefer the form to be like this, easier to maintain when inserting/removing/reorder the item

https://github.com/wikimedia/wikipedia-kaios/blob/86c635518ba873e06e00595666a572b8993f6fbb/src/components/AboutWikipedia.js#L13-L18